capnpy/testing/compiler/test_repr.py

```diff
@@ -42,6 +54,10 @@ def is_ascii(s):
             return
         capnp_repr = self.decode(obj)
         assert myrepr == capnp_repr, 'shortrepr() does not match with capnp decode'
+        #
+        new_obj = self.encode(obj.__class__, myrepr)
+        new_obj._print_buf()
```

Comment:
this is probably a debugging leftover which should be deleted?

Reply:
Well spotted, I've just removed it.